### PR TITLE
publish Docker image for non release builds as well

### PR DESCRIPTION
**Pull request by vladak**
This change should build and push `opengrok/docker:master` for each non release build in the main repo.

### CODE REVIEW

**Comment by vladak** [comment]
For testing, I already pushed `:master` to Docker hub by running the script with `DOCKER_USERNAME`, `DOCKER_PASSWORD` and `OPENGROK_REPO_SLUG` environment variables.

File: dev/docker.sh [hunk 1]
```diff
@@ -84,12 +81,12 @@ if [[ -z $DOCKER_PASSWORD ]]; then
 fi
 
 # Publish the image to Docker hub.
-if [ -n "$DOCKER_PASSWORD" -a -n "$DOCKER_USERNAME" -a -n "$VERSION" ]; then
+if [ -n "$DOCKER_PASSWORD" -a -n "$DOCKER_USERNAME" -a -n "$TAGS" ]; then
```
Comment (ahornace) [inline dev/docker.sh:84]: Isn't `-n "$TAGS"` always true?

**Comment by ahornace** [comment]
Just a nitpick I noticed, maybe we could remove the `Travis` reference as that is no longer true?
```
When pushing, this script uses the following Travis secure variables:
```

**Review by ahornace — APPROVED** [review]

File: dev/docker.sh (hunk 1 above)
Comment (vladak) [reply dev/docker.sh:84]: it is, however a bit of defensive programming does not hurt in shell scripts :-)

**Comment by vladak** [comment]
> Just a nitpick I noticed, maybe we could remove the `Travis` reference as that is no longer true?
> 
> ```
> When pushing, this script uses the following Travis secure variables:
> ```

True, removed.

**Pull request merged**